pkg/cmd/kafka/topic/create/create.go

```diff
@@ -280,10 +281,18 @@ func runInteractivePrompt(opts *Options) (err error) {
 		Default: fmt.Sprintf("%v", defaultRetentionPeriodMS),
 	}
 
-	err = survey.AskOne(retentionPrompt, &opts.retentionMs, survey.WithValidator(topic.ValidateMessageRetentionPeriod))
+	err = survey.AskOne(retentionPrompt, &opts.retentionMs, survey.WithValidator(topicutil.ValidateMessageRetentionPeriod))
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
+
+func createConfigEntries(opts *Options) *[]strimziadminclient.ConfigEntry {
```

Comment:
Should we use topicUtil's CreateConfigEntries method here as well?

Reply:
It does, see line 297.